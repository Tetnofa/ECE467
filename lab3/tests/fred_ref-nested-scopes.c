int a;

void f1() {
  bool a;
  int b;
  b = a;
}
