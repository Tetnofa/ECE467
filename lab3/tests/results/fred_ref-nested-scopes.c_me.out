sema: 6:2 : type mismatch 
